cli_reference/basic_cli_operations.adoc

```diff
@@ -107,6 +107,7 @@ syntax:
 |`build` |
 |`buildConfig` | `bc`
 |`deploymentConfig` | `dc`
+|`deployments` (Technology Preview)|
```

Comment:
OK thanks, since this PR is for 3.4, I'll add `deploy` as the abbreviation here.